scripts/lint.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#echo -n "Formatting golang code... "
+#gofmtout=$(go fmt ./...)
+#if [ "$gofmtout" ]; then
+#  printf "\n\n"
+#  echo "Files with formatting errors:"
+#  echo "${gofmtout}"
+#  exit 1
+#fi
+#echo "done."
+
+#echo -n "Linting golang code... "
```

Comment:
I want to enable formatting/linting in a separate PR since it will likely be a large diff that doesn't change any functionality.